### PR TITLE
HW8_Oleksandr_Petrov

### PR DESCRIPTION
**Pull request by PetrovAI-165**

### CODE REVIEW

File: Oleksadr_Petrov/HW8/Task2.py [hunk 1]
```diff
@@ -0,0 +1,13 @@
+def custom_filter(callback: callable, sequence: iter):
+    return [value for value in sequence if value == callback(value)]
+
+
+def even_numbers(number: int):
+    if number % 2 == 0:
+        return number
```
Comment (bohdandovbsyh) [inline Oleksadr_Petrov/HW8/Task2.py:7]: you can use lambda except for writing your own function. Just a suggestion, no need to remove it

File: Oleksadr_Petrov/HW8/Task2.py [hunk 2]
```diff
@@ -0,0 +1,13 @@
+def custom_filter(callback: callable, sequence: iter):
```
Comment (bohdandovbsyh) [inline Oleksadr_Petrov/HW8/Task2.py]: add return anotation

File: Oleksadr_Petrov/HW8/Task3.py [hunk 3]
```diff
@@ -0,0 +1,15 @@
+def custom_map(callback: callable, sequence: iter):
```
Comment (bohdandovbsyh) [inline Oleksadr_Petrov/HW8/Task3.py]: return annotation

File: Oleksadr_Petrov/HW8/Task4.py [hunk 4]
```diff
@@ -0,0 +1,61 @@
+def custom_max(collection: iter, results_count: int = 1):
```
Comment (bohdandovbsyh) [inline Oleksadr_Petrov/HW8/Task4.py]: return annotation

File: Oleksadr_Petrov/HW8/Task5.py [hunk 5]
```diff
@@ -0,0 +1,11 @@
+def custom_all(collection: iter):
```
Comment (bohdandovbsyh) [inline Oleksadr_Petrov/HW8/Task5.py]: return annotation

Comment (PetrovAI-165) [reply Oleksadr_Petrov/HW8/Task5.py]: resolved

File: Oleksadr_Petrov/HW8/Task4.py (hunk 4 above)
Comment (PetrovAI-165) [reply Oleksadr_Petrov/HW8/Task4.py]: resolved

File: Oleksadr_Petrov/HW8/Task3.py (hunk 3 above)
Comment (PetrovAI-165) [reply Oleksadr_Petrov/HW8/Task3.py]: resolved

File: Oleksadr_Petrov/HW8/Task2.py (hunk 2 above)
Comment (PetrovAI-165) [reply Oleksadr_Petrov/HW8/Task2.py]: resolved

File: Oleksadr_Petrov/HW8/Task2.py (hunk 1 above)
Comment (PetrovAI-165) [reply Oleksadr_Petrov/HW8/Task2.py:7]: the previous solution remained in the file, but additionally added an example with a lambda expression

**Review by bohdandovbsyh — APPROVED** [review]

**Pull request merged**